### DISCUSSION TITLE
How about that?

### DISCUSSION THREAD

**Discussion by NullTar** [localsend/localsend#530] [Discussion (English, International)] [answered]
Not localSend, should be localChat. I can send with localSend if the software developer need the design file, Its **free**

![Desktop1](https://github.com/localsend/localsend/assets/103525955/fd562010-b91b-42a6-9d3c-31badc419a1f)

![Desktop1-1](https://github.com/localsend/localsend/assets/103525955/b1cc5489-9a4d-4e8f-9b48-bbe02bb73040)

**Comment by Tienisto** [comment, marked as answer]
This is a good idea (and also mentioned in other issues), however, "remember" and "reconnect" to devices will be difficult.

The problem is that LocalSend has no external servers so devices only know the IP address of the other devices - and IP addresses are not constant.

**Reply by NullTar** [reply]
I got it! thank you, now I know it how to work. and I wanna chat cause there are a lot of beauty girls in my office. we like send files to each other and like send messages too, the only things is we can send a message once then we close the window do that around and around. but best thing is no records.

anyway, its a nice app! if you wanna re - design just come to me, totally free
